dev/src/builtins.cpp

```diff
@@ -710,7 +710,13 @@ nfr("normalize", "vec",  "F}" , "F}",
         VECTORVARS;
         VECTOROPNR(sql += f.fval() * f.fval());
         double m = sqrt(sql);
-        VECTOROPNR(elems[i] = f.fval() / m);
+        if (m == 0.0) {
+            for (iint i = 0; i < len; i++) {
```

Comment:
`VECTOROPNR(elems[i] = 0.0)` ?

Reply:
That's what I did at first, but the gcc/clang complained that ```/home/runner/work/lobster/lobster/dev/src/builtins.cpp:584:14: error: variable ‘f’ set but not used [-Werror=unused-but-set-variable]
  584 |         auto f = elems[i]; \```

Maybe `VECTOROPNR((void)f; elems[i] = 0.0)` if you prefer that?

Reply:
yeah, that's not great, but still better to be consistent.

Reply:
Done